### PR TITLE
misc-fixes

### PR DESCRIPTION
**Pull request by jacekkolasa**
various fixes for Agent Dashboard:
- fix registry when Do Not Track is enabled
- support whitespaces in replicated-nodes urls
- add trailing slash in replicated-nodes links
- disable loop to check for ACLK status (check just once, as in design)
- remove 1 level of nesting in Dashboard bundle (instead of lazy-loading, make alias in webpack based on env variable)
- add separate task to work with node-view
- describe available tasks, clean CRA readme

### CODE REVIEW

File: src/node-view-app.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  useEffect(() => {
+    axiosInstance.get("http://localhost:19999/api/v1/charts")
+      .then((response) => {
+        if (response.data) {
+          setRealMetadata(response.data)
+        }
+      })
+  }, [])
+
+  const [currentRoute, setCurrentRoute] = useState(() => getCurrentRouteFromHash())
```
Comment (builat) [inline src/node-view-app.tsx:57]:
```suggestion
  const [currentRoute, setCurrentRoute] = useState(getCurrentRouteFromHash)
```

File: src/node-view-app.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  return (
+    <div className="App" style={{ height: "100vh" }}>
```
Comment (builat) [inline src/node-view-app.tsx:60]: let's declare style object out of `return` and component, so we can avoid redundant object recreation

File: src/node-view-app.tsx (hunk 1 above)
Comment (jacekkolasa) [reply src/node-view-app.tsx]: yep

File: src/node-view-app.tsx (hunk 2 above)
Comment (jacekkolasa) [reply src/node-view-app.tsx]: sure, done

**Pull request merged**